### PR TITLE
add terms of service and acceptable use policy

### LINKED ISSUE #4511: Add TOS and AUP links to the openFEC swagger
**What we're after:** 
We need to include links to the Terms of Service and Acceptable Use Policy to the openFEC swagger. 

User story: As an API user, it's important that I understand if any limitations to using FEC data exist in order to use it effectively and understand potential blockers before I hit them. 

TOS: https://github.com/fecgov/FEC/blob/master/TERMS-OF-SERVICE.md
AUP: https://github.com/fecgov/FEC/blob/master/ACCEPTABLE-USE-POLICY.md

**Completion criteria:**
- [x] Updated language is drafted for the swagger page to include the TOS and AUP
- [x] Updated language is added to the swagger page

### PR DESCRIPTION
**Pull request by jason-upchurch**
## Summary (required)

- Resolves #4511

adds terms of service and acceptable use policy to the api key signup page (swagger-ui)

## How to test the changes locally
note to reviewers: 
required: @JonellaCulmer 
optional: @PaulClark2 @lbeaufort @johnnyporkchops 

- I'll [deploy to dev](https://fec-dev-api.app.cloud.gov/developers/) so @JonellaCulmer can review (as of 8/1/2020 1:30 p.m. it's available on dev)
- otherwise, run locally, review `localhost:5000`

Things to test:
- click links
- verify it looks as expected
- text was copy/paste other than formatting, but another read through may be good.

### CODE REVIEW

File: webservices/docs.py [hunk 1]
```diff
@@ -17,14 +17,10 @@
...
 
-If you would like to use the FEC's API programatically, you can sign up for
-your own API key below. But, you can still try out our API without
-an API key by using the web interface below and making use of a DEMO_KEY.
-Signing up for an API key will enable you to place up to 1,000 calls an hour.
-Each call is limited to 100 results per page. You can email questions, comments or
-a request to get a key for 120 calls per minute to [APIinfo@fec.gov](mailto:apiinfo@fec.gov). You can also
-ask questions and discuss the data in the [FEC data Google Group](https://groups.google.com/forum/#!forum/fec-data).
-API changes will also be added to this group in advance of the change.
+<h2>Getting started with the openFEC API</h2>
```
Comment (JonellaCulmer) [inline webservices/docs.py:20]: This new header just needs to be bolded text. @johnnyporkchops Does this look okay to you?

`<p>
<strong>Getting started with the openFEC API</strong>
</p>`

Comment (JonellaCulmer) [reply webservices/docs.py:20]: @johnnyporkchops Feel free to "edit" my comment to see the code I attempted.

**Comment by jason-upchurch** [comment]
@JonellaCulmer we don't want to keep it consistent with the other section you asked about? (So we would have a main title, then bold, then a larger section header after? Want to make sure that's what you believe will be most consistent.

Screen Shot 2020-07-31 at 4 12 36 PM

**Comment by jason-upchurch** [comment]
> t in there

Thanks! I replied https://github.com/fecgov/openFEC/pull/4532#issuecomment-667336536

I think it might look weird if we have a random bold in the same font and then an even bigger header afterwards, yeah?

File: webservices/docs.py [hunk 2]
```diff
@@ -17,14 +17,11 @@
...
-API changes will also be added to this group in advance of the change.
+<b class="body" id="getting_started_head">Getting started with the openFEC API</b><br>
+If you would like to use the FEC's API programmatically, you can sign up for your own API key using our form. Alternatively, you can still try out our API without an API key by using the web interface and using DEMO_KEY. Note that when you use the openFEC API you are subject to the [Terms of Service](https://github.com/fecgov/FEC/blob/master/TERMS-OF-SERVICE.md) and [Acceptable Use policy](https://github.com/fecgov/FEC/blob/master/ACCEPTABLE-USE-POLICY.md).
+
+
+Signing up for an API key will enable you to place up to 1,000 calls an hour. Each call is limited to 100 results per page. You can email questions, comments or a request to get a key for 7,600 calls per hour (120 calls per minute) to <a href="mailto:APIinfo@fec.gov">APIinfo@fec.gov</a>. You can also ask questions and discuss the data in the [FEC data Google Group](https://groups.google.com/forum/#!forum/fec-data). API changes will also be added to this group in advance of the change.
```
Comment (PaulClark2) [inline webservices/docs.py:24]: I’d like to edit this language, `You can also ask questions and discuss the data in the [FEC data Google Group](https://groups.google.com/forum/#!forum/fec-data). API changes will also be added to this group in advance of the change.`

I’d like to replace the first sentence with something like this, `You can also ask questions and discuss the data in a community led  [group](https://groups.google.com/forum/#!forum/fec-data).`

I’d like delete the second sentence, ~~API changes will also be added to this group in advance of the change.~~

Comment (PaulClark2) [reply webservices/docs.py:24]: We can still post API changes in the Google group, but I dint want us to be obligated to do it.

**Comment by jason-upchurch** [comment]
@PaulClark2 I've updated with your changes.

File: webservices/docs.py (hunk 2 above)
Comment (jason-upchurch) [reply webservices/docs.py]: updated

**Pull request merged**